components/editor/modules/link/ui.js

```diff
@@ -19,17 +20,37 @@ const getUsers = gql`
       lastName
       email
       id
+      portrait
     }
   }
 `
 
+const UserItem = ({ user }) => (
+  <div style={{ display: 'flex', alignItems: 'center' }}>
+    <img
+      src={user.portrait}
+      width='100%'
+      style={{ width: 54, marginRight: 10 }}
+    />
```

Comment:
I'd this align with [the frontend profile](https://github.com/orbiting/republik-frontend/blob/2753d6a152ed7499c81642b2a33b743d25b8e955/components/Profile/Portrait.js#L10-L17) and display an gray area when there is none.
```suggestion
    <div
      style={{
        width: 54,
        height: 54,
        backgroundColor: '#E2E8E6',
        backgroundImage: user.portrait ? `url(${user.portrait})` : undefined,
        backgroundSize: 'cover',
        backgroundPosition: 'center',
        marginRight: 10
      }}
    />
```